tools/distpartitioning/data_shuffle.py

```diff
@@ -11,17 +11,17 @@
 from timeit import default_timer as timer
 from datetime import timedelta
 from dataset_utils import get_dataset
-from utils import read_partitions_file, read_json, get_node_types, \
+from utils import read_ntype_partition_files, read_json, get_node_types, \
                     augment_edge_data, get_gnid_range_map, \
-                    write_dgl_objects, write_metadata_json
+                    write_dgl_objects, write_metadata_json, get_ntype_featnames
 from gloo_wrapper import alltoall_cpu_object_lst, alltoallv_cpu, \
                     alltoall_cpu, allgather_sizes, gather_metadata_json
 from globalids import assign_shuffle_global_nids_nodes, \
                     assign_shuffle_global_nids_edges, \
                     get_shuffle_global_nids_edges
 from convert_partition import create_dgl_object, create_metadata_json
 
-def gen_node_data(rank, world_size, node_part_ids, ntid_ntype_map, nid_schema_map):
+def gen_node_data(rank, world_size, node_part_ids, ntid_ntype_map, schema_map):
```

Comment:
`schema_map` is changed. you should update the docstring.

Reply:
Agreed. Updated the docstring to match with the current updated file format. Also explained how this information can be extracted from the metadata json file.